### PR TITLE
Fix QML build regressions on macOS

### PR DESCRIPTION
**Pull request by fwcd**
This fixes a few minor regressions introduced by #11407 when building with `-DQML=ON` on macOS and re-enables QML in the CI build for macOS.

### CODE REVIEW

File: src/controllers/scripting/legacy/controllerscriptenginelegacy.cpp
Comment (Swiftb0y) [inline src/controllers/scripting/legacy/controllerscriptenginelegacy.cpp:331]: Judging by the fact that `std::execution` seems very much broken still lets remove it outright. In that case the weird `#pragma push_macro` at the top of the file can also be removed. wdyt?

Comment (fwcd) [reply src/controllers/scripting/legacy/controllerscriptenginelegacy.cpp:331]: Sounds good to me, wdyt @acolombier?

Comment (acolombier) [reply src/controllers/scripting/legacy/controllerscriptenginelegacy.cpp:331]: Yep, sounds good to me. 
@Swiftb0y wanted me to put so if he is happy to remove, I'm happy too! :D

**Review by acolombier — APPROVED** [review]
Thanks for the quick fix!

My fault for not testing On MacOS after doing all the nits addressing on that beast PR

**Comment by fwcd** [comment]
Hm, it looks like `std::bit_cast` might be unavailable too on the older macOS 11 runners we're building on (see https://github.com/mixxxdj/mixxx/actions/runs/9402107656/job/25895415853). This seems to have been implemented/stabilized, at least I couldn't reproduce it with Xcode 15 on macOS 14, but I don't know how that would affect backwards compatibility here.

File: src/controllers/scripting/legacy/controllerscriptenginelegacy.cpp
Comment (fwcd) [inline src/controllers/scripting/legacy/controllerscriptenginelegacy.cpp:657]: From what I understand, casting between `const char *` and `const unsigned char *` using `reinterpret_cast` seems to be fine, [stackoverflow confirms that](https://stackoverflow.com/questions/28258136/using-a-pointer-for-casting-char-to-unsigned-char). This would fix the build on some older compilers where `std::bit_cast` isn't available, such as the macOS 11 runners we are building on in CI.

Would appreciate a second pair of eyes checking that this is fine though.

Comment (acolombier) [inline src/controllers/scripting/legacy/controllerscriptenginelegacy.cpp:657]: AFAIR, this is what I was using, but I was requested to change it to `bit_cast`. Perhaps you could keep this cast specific to Mac?

Comment (acolombier) [reply src/controllers/scripting/legacy/controllerscriptenginelegacy.cpp:657]: A comment sounds good yeah!

Comment (Swiftb0y) [reply src/controllers/scripting/legacy/controllerscriptenginelegacy.cpp:659]: > Perhaps a comment that this should be refactored to a std::bit_cast once we drop support for macOS 11 (or build on a newer runner)?

is this specific to the build environment or to the runner? Cuz the runner was recently updated #13296. 

Here's a little more context for reinterpret_cast vs std::bit_cast: https://stackoverflow.com/a/53402217

Comment (fwcd) [reply src/controllers/scripting/legacy/controllerscriptenginelegacy.cpp:659]: Should be specific to the compiler, i.e. the Xcode version that the runner uses. macOS 12 is two years old too, so my best guess would be that it doesn't support bit_cast either yet.

Comment (Swiftb0y) [reply src/controllers/scripting/legacy/controllerscriptenginelegacy.cpp:659]: https://developer.apple.com/xcode/cpp/ says bit-casting requires XCode 14.3. Seems like that's only available on MacOS 13...

**Review by Swiftb0y — APPROVED** [review]
LGTM. waiting for CI. Thank you.

File: src/controllers/scripting/legacy/controllerscriptenginelegacy.cpp
Comment (fwcd) [reply src/controllers/scripting/legacy/controllerscriptenginelegacy.cpp:659]: Ah okay, thanks for checking. I think as long as we're building against an SDK older than that (in CI), we are going to get an error, regardless of whether we use a newer runner. I'll update the comment.

Comment (fwcd) [reply src/controllers/scripting/legacy/controllerscriptenginelegacy.cpp:659]: That's a neat site btw, have to bookmark that.

Comment (Swiftb0y) [reply src/controllers/scripting/legacy/controllerscriptenginelegacy.cpp:659]: Even better: https://en.cppreference.com/w/cpp/compiler_support

Apparently they also track apple clang now, which they previously haven't which is why I defaulted to the apple site.

**Comment by fwcd** [comment]
Any ideas on the test failure? That it only fails on x64 makes it pretty strange

**Comment by acolombier** [comment]
Yep, that's the part that is failing:
```
warning [0x7fc842d082d0] Unable to load the QML scene: QUrl("file:///Users/runner/work/mixxx/mixxx/res/qml/DummyDeviceDefaultScreen.qml") at line 9 , error:  file:///Users/runner/work/mixxx/mixxx/res/qml/DummyDeviceDefaultScreen.qml:9:1: module "Qt5Compat.GraphicalEffects" is not installed
warning [0x7fc842d082d0] ControllerScriptHandlerBase: "Uncaught exception: /Users/runner/work/mixxx/mixxx/res/qml/DummyDeviceDefaultScreen.qml:9: module \"Qt5Compat.GraphicalEffects\" is not installed"
warning [0x7fc842d082d0] Unable to load the QML scene: QUrl("file:///Users/runner/work/mixxx/mixxx/res/qml/DummyDeviceDefaultScreen.qml") at line 7 , error:  file:///Users/runner/work/mixxx/mixxx/res/qml/DummyDeviceDefaultScreen.qml:7:1: module "QtQuick.Window" is not installed
warning [0x7fc842d082d0] ControllerScriptHandlerBase: "Uncaught exception: /Users/runner/work/mixxx/mixxx/res/qml/DummyDeviceDefaultScreen.qml:7: module \"QtQuick.Window\" is not installed"
warning [0x7fc842d082d0] Unable to load the QML scene: QUrl("file:///Users/runner/work/mixxx/mixxx/res/qml/DummyDeviceDefaultScreen.qml") at line -1 , error:  file:///Users/runner/work/mixxx/mixxx/res/qml/DummyDeviceDefaultScreen.qml: module "QtQuick.Controls.macOS" is not installed
warning [0x7fc842d082d0] ControllerScriptHandlerBase: "Uncaught exception: /Users/runner/work/mixxx/mixxx/res/qml/DummyDeviceDefaultScreen.qml:-1: module \"QtQuick.Controls.macOS\" is not installed"
warning [0x7fc842d082d0] Unable to load the QML scene: QUrl("file:///Users/runner/work/mixxx/mixxx/res/qml/DummyDeviceDefaultScreen.qml") at line 2 , error:  file:///Users/runner/work/mixxx/mixxx/res/qml/DummyDeviceDefaultScreen.qml:2:1: module "QtQuick.Window" is not installed
warning [0x7fc842d082d0] ControllerScriptHandlerBase: "Uncaught exception: /Users/runner/work/mixxx/mixxx/res/qml/DummyDeviceDefaultScreen.qml:2: module \"QtQuick.Window\" is not installed"
warning [0x7fc842d082d0] Unable to load the QML scene: QUrl("file:///Users/runner/work/mixxx/mixxx/res/qml/DummyDeviceDefaultScreen.qml") at line 9 , error:  file:///Users/runner/work/mixxx/mixxx/res/qml/DummyDeviceDefaultScreen.qml:9:1: module "Qt5Compat.GraphicalEffects" is not installed
warning [0x7fc842d082d0] ControllerScriptHandlerBase: "Uncaught exception: /Users/runner/work/mixxx/mixxx/res/qml/DummyDeviceDefaultScreen.qml:9: module \"Qt5Compat.GraphicalEffects\" is not installed"
warning [0x7fc842d082d0] Unable to load the QML scene: QUrl("file:///Users/runner/work/mixxx/mixxx/res/qml/DummyDeviceDefaultScreen.qml") at line 7 , error:  file:///Users/runner/work/mixxx/mixxx/res/qml/DummyDeviceDefaultScreen.qml:7:1: module "QtQuick.Window" is not installed
warning [0x7fc842d082d0] ControllerScriptHandlerBase: "Uncaught exception: /Users/runner/work/mixxx/mixxx/res/qml/DummyDeviceDefaultScreen.qml:7: module \"QtQuick.Window\" is not installed"
warning [0x7fc842d082d0] Unable to load the QML scene: QUrl("file:///Users/runner/work/mixxx/mixxx/res/qml/DummyDeviceDefaultScreen.qml") at line -1 , error:  file:///Users/runner/work/mixxx/mixxx/res/qml/DummyDeviceDefaultScreen.qml: module "QtQuick.Controls.macOS" is not installed
warning [0x7fc842d082d0] ControllerScriptHandlerBase: "Uncaught exception: /Users/runner/work/mixxx/mixxx/res/qml/DummyDeviceDefaultScreen.qml:-1: module \"QtQuick.Controls.macOS\" is not installed"
warning [0x7fc842d082d0] Unable to load the QML scene: QUrl("file:///Users/runner/work/mixxx/mixxx/res/qml/DummyDeviceDefaultScreen.qml") at line 2 , error:  file:///Users/runner/work/mixxx/mixxx/res/qml/DummyDeviceDefaultScreen.qml:2:1: module "QtQuick.Window" is not installed
warning [0x7fc842d082d0] ControllerScriptHandlerBase: "Uncaught exception: /Users/runner/work/mixxx/mixxx/res/qml/DummyDeviceDefaultScreen.qml:2: module \"QtQuick.Window\" is not installed"
warning [0x7fc842d082d0] Unable to load the QML scene: QUrl("file:///Users/runner/work/mixxx/mixxx/res/qml/DummyDeviceDefaultScreen.qml") at line 9 , error:  file:///Users/runner/work/mixxx/mixxx/res/qml/DummyDeviceDefaultScreen.qml:9:1: module "Qt5Compat.GraphicalEffects" is not installed
warning [0x7fc842d082d0] ControllerScriptHandlerBase: "Uncaught exception: /Users/runner/work/mixxx/mixxx/res/qml/DummyDeviceDefaultScreen.qml:9: module \"Qt5Compat.GraphicalEffects\" is not installed"
warning [0x7fc842d082d0] Unable to load the QML scene: QUrl("file:///Users/runner/work/mixxx/mixxx/res/qml/DummyDeviceDefaultScreen.qml") at line 7 , error:  file:///Users/runner/work/mixxx/mixxx/res/qml/DummyDeviceDefaultScreen.qml:7:1: module "QtQuick.Window" is not installed
warning [0x7fc842d082d0] ControllerScriptHandlerBase: "Uncaught exception: /Users/runner/work/mixxx/mixxx/res/qml/DummyDeviceDefaultScreen.qml:7: module \"QtQuick.Window\" is not installed"
warning [0x7fc842d082d0] Unable to load the QML scene: QUrl("file:///Users/runner/work/mixxx/mixxx/res/qml/DummyDeviceDefaultScreen.qml") at line -1 , error:  file:///Users/runner/work/mixxx/mixxx/res/qml/DummyDeviceDefaultScreen.qml: module "QtQuick.Controls.macOS" is not installed
warning [0x7fc842d082d0] ControllerScriptHandlerBase: "Uncaught exception: /Users/runner/work/mixxx/mixxx/res/qml/DummyDeviceDefaultScreen.qml:-1: module \"QtQuick.Controls.macOS\" is not installed"
warning [0x7fc842d082d0] Unable to load the QML scene: QUrl("file:///Users/runner/work/mixxx/mixxx/res/qml/DummyDeviceDefaultScreen.qml") at line 2 , error:  file:///Users/runner/work/mixxx/mixxx/res/qml/DummyDeviceDefaultScreen.qml:2:1: module "QtQuick.Window" is not installed
warning [0x7fc842d082d0] ControllerScriptHandlerBase: "Uncaught exception: /Users/runner/work/mixxx/mixxx/res/qml/DummyDeviceDefaultScreen.qml:2: module \"QtQuick.Window\" is not installed"
warning [0x7fc842d082d0] Unable to load the QML scene: QUrl("file:///Users/runner/work/mixxx/mixxx/res/qml/DummyDeviceDefaultScreen.qml") at line 9 , error:  file:///Users/runner/work/mixxx/mixxx/res/qml/DummyDeviceDefaultScreen.qml:9:1: module "Qt5Compat.GraphicalEffects" is not installed
warning [0x7fc842d082d0] ControllerScriptHandlerBase: "Uncaught exception: /Users/runner/work/mixxx/mixxx/res/qml/DummyDeviceDefaultScreen.qml:9: module \"Qt5Compat.GraphicalEffects\" is not installed"
warning [0x7fc842d082d0] Unable to load the QML scene: QUrl("file:///Users/runner/work/mixxx/mixxx/res/qml/DummyDeviceDefaultScreen.qml") at line 7 , error:  file:///Users/runner/work/mixxx/mixxx/res/qml/DummyDeviceDefaultScreen.qml:7:1: module "QtQuick.Window" is not installed
warning [0x7fc842d082d0] ControllerScriptHandlerBase: "Uncaught exception: /Users/runner/work/mixxx/mixxx/res/qml/DummyDeviceDefaultScreen.qml:7: module \"QtQuick.Window\" is not installed"
warning [0x7fc842d082d0] Unable to load the QML scene: QUrl("file:///Users/runner/work/mixxx/mixxx/res/qml/DummyDeviceDefaultScreen.qml") at line -1 , error:  file:///Users/runner/work/mixxx/mixxx/res/qml/DummyDeviceDefaultScreen.qml: module "QtQuick.Controls.macOS" is not installed
warning [0x7fc842d082d0] ControllerScriptHandlerBase: "Uncaught exception: /Users/runner/work/mixxx/mixxx/res/qml/DummyDeviceDefaultScreen.qml:-1: module \"QtQuick.Controls.macOS\" is not installed"
warning [0x7fc842d082d0] Unable to load the QML scene: QUrl("file:///Users/runner/work/mixxx/mixxx/res/qml/DummyDeviceDefaultScreen.qml") at line 2 , error:  file:///Users/runner/work/mixxx/mixxx/res/qml/DummyDeviceDefaultScreen.qml:2:1: module "QtQuick.Window" is not installed
warning [0x7fc842d082d0] ControllerScriptHandlerBase: "Uncaught exception: /Users/runner/work/mixxx/mixxx/res/qml/DummyDeviceDefaultScreen.qml:2: module \"QtQuick.Window\" is not installed"
warning [0x7fc842d082d0] Unable to load the QML scene: QUrl("file:///Users/runner/work/mixxx/mixxx/res/qml/DummyDeviceDefaultScreen.qml") at line 9 , error:  file:///Users/runner/work/mixxx/mixxx/res/qml/DummyDeviceDefaultScreen.qml:9:1: module "Qt5Compat.GraphicalEffects" is not installed
warning [0x7fc842d082d0] ControllerScriptHandlerBase: "Uncaught exception: /Users/runner/work/mixxx/mixxx/res/qml/DummyDeviceDefaultScreen.qml:9: module \"Qt5Compat.GraphicalEffects\" is not installed"
warning [0x7fc842d082d0] Unable to load the QML scene: QUrl("file:///Users/runner/work/mixxx/mixxx/res/qml/DummyDeviceDefaultScreen.qml") at line 7 , error:  file:///Users/runner/work/mixxx/mixxx/res/qml/DummyDeviceDefaultScreen.qml:7:1: module "QtQuick.Window" is not installed
warning [0x7fc842d082d0] ControllerScriptHandlerBase: "Uncaught exception: /Users/runner/work/mixxx/mixxx/res/qml/DummyDeviceDefaultScreen.qml:7: module \"QtQuick.Window\" is not installed"
warning [0x7fc842d082d0] Unable to load the QML scene: QUrl("file:///Users/runner/work/mixxx/mixxx/res/qml/DummyDeviceDefaultScreen.qml") at line -1 , error:  file:///Users/runner/work/mixxx/mixxx/res/qml/DummyDeviceDefaultScreen.qml: module "QtQuick.Controls.macOS" is not installed
warning [0x7fc842d082d0] ControllerScriptHandlerBase: "Uncaught exception: /Users/runner/work/mixxx/mixxx/res/qml/DummyDeviceDefaultScreen.qml:-1: module \"QtQuick.Controls.macOS\" is not installed"
warning [0x7fc842d082d0] Unable to load the QML scene: QUrl("file:///Users/runner/work/mixxx/mixxx/res/qml/DummyDeviceDefaultScreen.qml") at line 2 , error:  file:///Users/runner/work/mixxx/mixxx/res/qml/DummyDeviceDefaultScreen.qml:2:1: module "QtQuick.Window" is not installed
warning [0x7fc842d082d0] ControllerScriptHandlerBase: "Uncaught exception: /Users/runner/work/mixxx/mixxx/res/qml/DummyDeviceDefaultScreen.qml:2: module \"QtQuick.Window\" is not installed"
warning [0x7fc842d082d0] Unable to load the QML scene: QUrl("file:///Users/runner/work/mixxx/mixxx/res/qml/DummyDeviceDefaultScreen.qml") at line 9 , error:  file:///Users/runner/work/mixxx/mixxx/res/qml/DummyDeviceDefaultScreen.qml:9:1: module "Qt5Compat.GraphicalEffects" is not installed
warning [0x7fc842d082d0] ControllerScriptHandlerBase: "Uncaught exception: /Users/runner/work/mixxx/mixxx/res/qml/DummyDeviceDefaultScreen.qml:9: module \"Qt5Compat.GraphicalEffects\" is not installed"
warning [0x7fc842d082d0] Unable to load the QML scene: QUrl("file:///Users/runner/work/mixxx/mixxx/res/qml/DummyDeviceDefaultScreen.qml") at line 7 , error:  file:///Users/runner/work/mixxx/mixxx/res/qml/DummyDeviceDefaultScreen.qml:7:1: module "QtQuick.Window" is not installed
warning [0x7fc842d082d0] ControllerScriptHandlerBase: "Uncaught exception: /Users/runner/work/mixxx/mixxx/res/qml/DummyDeviceDefaultScreen.qml:7: module \"QtQuick.Window\" is not installed"
warning [0x7fc842d082d0] Unable to load the QML scene: QUrl("file:///Users/runner/work/mixxx/mixxx/res/qml/DummyDeviceDefaultScreen.qml") at line -1 , error:  file:///Users/runner/work/mixxx/mixxx/res/qml/DummyDeviceDefaultScreen.qml: module "QtQuick.Controls.macOS" is not installed
warning [0x7fc842d082d0] ControllerScriptHandlerBase: "Uncaught exception: /Users/runner/work/mixxx/mixxx/res/qml/DummyDeviceDefaultScreen.qml:-1: module \"QtQuick.Controls.macOS\" is not installed"
warning [0x7fc842d082d0] Unable to load the QML scene: QUrl("file:///Users/runner/work/mixxx/mixxx/res/qml/DummyDeviceDefaultScreen.qml") at line 2 , error:  file:///Users/runner/work/mixxx/mixxx/res/qml/DummyDeviceDefaultScreen.qml:2:1: module "QtQuick.Window" is not installed
warning [0x7fc842d082d0] ControllerScriptHandlerBase: "Uncaught exception: /Users/runner/work/mixxx/mixxx/res/qml/DummyDeviceDefaultScreen.qml:2: module \"QtQuick.Window\" is not installed"
warning [0x7fc842d082d0] Unable to load the QML scene: QUrl("file:///Users/runner/work/mixxx/mixxx/res/qml/DummyDeviceDefaultScreen.qml") at line 9 , error:  file:///Users/runner/work/mixxx/mixxx/res/qml/DummyDeviceDefaultScreen.qml:9:1: module "Qt5Compat.GraphicalEffects" is not installed
warning [0x7fc842d082d0] ControllerScriptHandlerBase: "Uncaught exception: /Users/runner/work/mixxx/mixxx/res/qml/DummyDeviceDefaultScreen.qml:9: module \"Qt5Compat.GraphicalEffects\" is not installed"
warning [0x7fc842d082d0] Unable to load the QML scene: QUrl("file:///Users/runner/work/mixxx/mixxx/res/qml/DummyDeviceDefaultScreen.qml") at line 7 , error:  file:///Users/runner/work/mixxx/mixxx/res/qml/DummyDeviceDefaultScreen.qml:7:1: module "QtQuick.Window" is not installed
warning [0x7fc842d082d0] ControllerScriptHandlerBase: "Uncaught exception: /Users/runner/work/mixxx/mixxx/res/qml/DummyDeviceDefaultScreen.qml:7: module \"QtQuick.Window\" is not installed"
warning [0x7fc842d082d0] Unable to load the QML scene: QUrl("file:///Users/runner/work/mixxx/mixxx/res/qml/DummyDeviceDefaultScreen.qml") at line -1 , error:  file:///Users/runner/work/mixxx/mixxx/res/qml/DummyDeviceDefaultScreen.qml: module "QtQuick.Controls.macOS" is not installed
warning [0x7fc842d082d0] ControllerScriptHandlerBase: "Uncaught exception: /Users/runner/work/mixxx/mixxx/res/qml/DummyDeviceDefaultScreen.qml:-1: module \"QtQuick.Controls.macOS\" is not installed"
warning [0x7fc842d082d0] Unable to load the QML scene: QUrl("file:///Users/runner/work/mixxx/mixxx/res/qml/DummyDeviceDefaultScreen.qml") at line 2 , error:  file:///Users/runner/work/mixxx/mixxx/res/qml/DummyDeviceDefaultScreen.qml:2:1: module "QtQuick.Window" is not installed
warning [0x7fc842d082d0] ControllerScriptHandlerBase: "Uncaught exception: /Users/runner/work/mixxx/mixxx/res/qml/DummyDeviceDefaultScreen.qml:2: module \"QtQuick.Window\" is not installed"
warning [0x7fc842d082d0] Unable to load the QML scene: QUrl("file:///Users/runner/work/mixxx/mixxx/res/qml/DummyDeviceDefaultScreen.qml") at line 9 , error:  file:///Users/runner/work/mixxx/mixxx/res/qml/DummyDeviceDefaultScreen.qml:9:1: module "Qt5Compat.GraphicalEffects" is not installed
warning [0x7fc842d082d0] ControllerScriptHandlerBase: "Uncaught exception: /Users/runner/work/mixxx/mixxx/res/qml/DummyDeviceDefaultScreen.qml:9: module \"Qt5Compat.GraphicalEffects\" is not installed"
warning [0x7fc842d082d0] Unable to load the QML scene: QUrl("file:///Users/runner/work/mixxx/mixxx/res/qml/DummyDeviceDefaultScreen.qml") at line 7 , error:  file:///Users/runner/work/mixxx/mixxx/res/qml/DummyDeviceDefaultScreen.qml:7:1: module "QtQuick.Window" is not installed
warning [0x7fc842d082d0] ControllerScriptHandlerBase: "Uncaught exception: /Users/runner/work/mixxx/mixxx/res/qml/DummyDeviceDefaultScreen.qml:7: module \"QtQuick.Window\" is not installed"
warning [0x7fc842d082d0] Unable to load the QML scene: QUrl("file:///Users/runner/work/mixxx/mixxx/res/qml/DummyDeviceDefaultScreen.qml") at line -1 , error:  file:///Users/runner/work/mixxx/mixxx/res/qml/DummyDeviceDefaultScreen.qml: module "QtQuick.Controls.macOS" is not installed
warning [0x7fc842d082d0] ControllerScriptHandlerBase: "Uncaught exception: /Users/runner/work/mixxx/mixxx/res/qml/DummyDeviceDefaultScreen.qml:-1: module \"QtQuick.Controls.macOS\" is not installed"
warning [0x7fc842d082d0] Unable to load the QML scene: QUrl("file:///Users/runner/work/mixxx/mixxx/res/qml/DummyDeviceDefaultScreen.qml") at line 2 , error:  file:///Users/runner/work/mixxx/mixxx/res/qml/DummyDeviceDefaultScreen.qml:2:1: module "QtQuick.Window" is not installed
warning [0x7fc842d082d0] ControllerScriptHandlerBase: "Uncaught exception: /Users/runner/work/mixxx/mixxx/res/qml/DummyDeviceDefaultScreen.qml:2: module \"QtQuick.Window\" is not installed"
info [0x7fc842d082d0]   Shutting down engine
./src/test/controller_mapping_validation_test.cpp:261: Failure
```

This is likely because the QML dependencies aren't included in the VCPKG, or aren't installed properly in the build environment, basically meaning that QML doesn't work on Mixxx out of the box. This was tracked in #12267 and should have been fixed in #12604 but not sure this was tested on Mac x64

**Comment by fwcd** [comment]
Hm okay, so many disabling QML in x64 CI would be a temporary fix, not ideal, but at least we'd get QML CI on macOS.

**Comment by acolombier** [comment]
Do you mind creating an issue for macOS x64 /w QML just so we keep track of this?

**Comment by fwcd** [comment]
-> #13336

**Review by Swiftb0y — APPROVED** [review]
Welp, at least QML on arm macos is better than not on macos at all.

**Comment by Swiftb0y** [comment]
Thank you for looking into the issue this quickly.

**Pull request merged**
